lang/test/org/partiql/lang/syntax/SqlParserDateTimeTests.kt

```diff
@@ -172,10 +163,32 @@ class SqlParserDateTimeTests : SqlParserTestBase() {
         },
         DateTimeTestCase("TIME (0) WITH TIME ZONE '23:59:59.123456789-18:00'") {
             litTime(timeValue(23, 59, 59, 123456789, 0, -1080))
-        },
-        DateTimeTestCase("TIME (10) WITH TIME ZONE '23:59:59.123456789+18:00'") {
-            litTime(timeValue(23, 59, 59, 123456789, 9, 1080))
         }
+        // TODO: These tests should pass. Check https://github.com/partiql/partiql-lang-kotlin/issues/395
+//        DateTimeTestCase("TIME '23:59:59.1234567890'") {
+//            litTime(timeValue(23, 59, 59, 123456789, 9, null))
+//        },
+//        DateTimeTestCase("TIME '23:59:59.1234567899'") {
+//            litTime(timeValue(23, 59, 59, 123456790, 9, null))
+//        },
+//        DateTimeTestCase("TIME '23:59:59.1234567890+18:00'") {
+//            litTime(timeValue(23, 59, 59, 123456789, 9, null))
+//        },
```

Comment:
Are we able to have these uncommented and somehow ignored (might be a limitation of test framework)?

Reply:
Not sure if I can `@ignore` these. I have kept those commented to keep track of the issue we want to resolve later.

Reply:
The only way is to add a `skip` property to `DateTimeTestCase` and then filter out those cases where `testCase.skip == false`, which could easily be done.